flavors/standard-EXASOL-all-java-11/flavor_base/flavor_base_deps/packages/apt_get_packages

```diff
@@ -0,0 +1,3 @@
+coreutils|8.32-4.1ubuntu1.2
+locales|2.35-0ubuntu3.9
+wget|1.21.2-2ubuntu1.1
```

Comment:
We already have curls, so wget is probably optional

Reply:
Removed it